pandas/tests/groupby/transform/test_transform.py

```diff
@@ -357,7 +362,11 @@ def test_transform_transformation_func(request, transformation_func):
         },
         index=date_range("2020-01-01", "2020-01-07"),
     )
-
+    # TODO(2.0) Remove after pad/backfill deprecation enforced
```

Comment:
instead of this can we not just remove these from the list of all functions (and then put them in a separate list where we ignore the warnings)?

Reply:
We could remove from the list and put in a new list, but testing the new list and ignoring the warnings would involve more code than these `if/else` statements.

Reply:
ok can you make a function to do this then just to avoid the code bloat